File: inf1000/uke1/Handletur.java
/**
1.12: Handletur
*/

import java.util.Scanner;

class Handletur {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        final int BROD = 20; //kr
        final int MELK = 15;
        final int OST = 40;
        final int YOUGHURT = 12;

        System.out.print("Antall brod: ");
        int antBrod = Integer.parseInt(in.next());

        System.out.print("Antall melk: ");
        int antMelk = Integer.parseInt(in.next());

        System.out.print("Antall ost: ");
        int antOst = Integer.parseInt(in.next());

        System.out.print("Antall youghurt: ");
        int antYoughurt = Integer.parseInt(in.next());

        int sumBrod = BROD*antBrod;
        int sumMelk = MELK*antMelk;
        int sumOst = OST*antOst;
        int sumYoughurt = YOUGHURT*antYoughurt;

        int totalSum = sumBrod+sumMelk+sumOst+sumYoughurt;

        System.out.println("Handleturen kostet: " + totalSum + " kr.");
    }
}

File: inf1001/uke2/handletur.py
# Oppgave 2.10: Handletur

BROD = 20 #kr
MELK = 15
OST = 40
YOUGHURT = 12

ant_brod = input("Skriv inn antall brod: ")
ant_melk = input("Skriv inn antall melk: ")
ant_ost = input("Skriv inn antall ost: ")
ant_youghurt = input("Skriv inn antall youghurt: ")

tot_sum = ant_brod*BROD + ant_melk*MELK + ant_ost*OST + ant_youghurt*YOUGHURT
print "Handleturen kostet: ", tot_sum, "kr."
